org-super-agenda.el

```diff
@@ -272,12 +272,15 @@ face `org-super-agenda-header' appended, and the text properties
                           (string org-super-agenda-header-separator))))
          (setq s (concat " " s))
          (add-face-text-property 0 (length s) 'org-super-agenda-header t s)
-         (org-add-props s nil
+         (org-add-props s nil 'face 'org-super-agenda-header
            'keymap org-super-agenda-header-map
            ;; NOTE: According to the manual, only `keymap' should be necessary, but in my
            ;; testing, it only takes effect in Agenda buffers when `local-map' is set, so
            ;; we'll use both.
-           'local-map org-super-agenda-header-map)
+           'local-map org-super-agenda-header-map
+           'org-super-agenda-header t)
+         (org-add-props org-super-agenda-header-separator nil
```

Comment:
It's probably not quite correct to add this property to this special variable every time this function is called.  I think the property should be applied to `separator` instead.  However, see next comment...